src/js/main.js

```
@@ -1,3 +1,21 @@
+// View<->model bridge for the #incselect dropdown. The dropdown packs two server
+// fields (a numeric increment + a boolean increment_scales) into one option token
+// like "20" (fixed) or "1*n" (scales with move number). Increment is treated as
+// numeric DATA everywhere; the string token only exists as the <select>.value.
+
+// Decode an #incselect token into { increment: Number, increment_scales: Boolean }.
+function parseIncrementValue(value) {
+	const str = String(value ?? "0");
+	const scales = str.endsWith("*n");
+	const increment = parseInt(scales ? str.slice(0, -2) : str, 10) || 0;
+	return { increment, increment_scales: scales };
+}
```

ESLint

[error] Unexpected space before opening brace.

(space-before-blocks)